### PR TITLE
En 338 user profile management

### PR DESCRIPTION
**Pull request by stefanopuddu**
Contains basic junit test for Entity Managers
- UserProfileManager (En-338)
- DataObjectManager (En-335)
- ContentManager (En-340)

### CODE REVIEW

**Comment by entando-service** [comment]
Something went wrong. Investigate!

File: engine/src/main/java/com/agiletec/aps/system/common/entity/ApsEntityManager.java [hunk 1]
```diff
@@ -52,33 +53,61 @@
...
-	 * @throws Exception If errors are detected during the service initialization.
+	 * Prefix of the thread used for references reloading.
 	 */
+	public static final String RELOAD_REFERENCES_THREAD_NAME_PREFIX = "RELOAD_REFERENCES_INDEX_";
+
+	private static final Logger _logger = LoggerFactory.getLogger(ApsEntityManager.class);
+
+	private IEntityTypeFactory _entityTypeFactory;
+
+	private Class _entityClass;
```
Comment (joewhite101) [inline engine/src/main/java/com/agiletec/aps/system/common/entity/ApsEntityManager.java]: No underscores in new code. We should update to track java standard naming

File: engine/src/main/java/com/agiletec/aps/system/common/entity/cache/EntityManagerCacheWrapper.java [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.cache.Cache;
+
+/**
+ * @author E.Santoboni
+ */
+public class EntityManagerCacheWrapper extends AbstractCacheWrapper implements IEntityManagerCacheWrapper {
+
+	private static final Logger _logger = LoggerFactory.getLogger(EntityManagerCacheWrapper.class);
```
Comment (joewhite101) [inline engine/src/main/java/com/agiletec/aps/system/common/entity/cache/EntityManagerCacheWrapper.java]: Remove underscore.

**Comment by entando-service** [comment]
All is well

**Comment by entando-service** [comment]
All is well

**Pull request merged**